week-1/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,13 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function shout() {
+  var x = ["Daniel", "Irina", "Mimi", "Rob", "Yohannes"];
```

Comment:
Generally, typing out things twice is dangerous - what if you make a typo?
What can you do here instead of writing out strings? :)

Reply:
OK. I will.